packages/visualizations-react/src/reactify/ReactImpl.tsx

```diff
@@ -5,7 +5,8 @@ import './reactify.css';
 
 function reactifySvelte<P extends Record<string, unknown>>(
     Component: new (options: ComponentConstructorOptions) => SvelteComponent,
-    className: string
+    className: string,
+    isVisualization = true
```

Comment:
Now that the SDK exposes `LinksMenu.svelte` and it is not a visualization, I added the `isVisualization` prop to distinguish it from the other exported components, so that the `ods-visualization` class is not added to the root element.